source/fe/fe_enriched.cc

```diff
@@ -1258,13 +1258,13 @@ namespace ColorEnriched
        * Each time we build constraints at the
        * interface between two different FE_Enriched, we look for the least
        * dominating FE via
-       * hp::FECollection< dim, spacedim >::find_least_face_dominating_fe().
-       * If we don't take further actions, we may find a dominating FE that
-       * is too restrictive, i.e. enriched FE consisting of only FE_Nothing.
-       * New elements needs to be added to FECollection object to help
-       * find the correct enriched FE underlying the spaces in the
-       * adjacent cells. This is done by creating an appropriate
-       * set in fe_sets and a call to the function
+       * hp::FECollection< dim, spacedim
+       * >::find_least_face_dominating_fe_in_collection(). If we don't take
```

Comment:
see if you can break this better so that the closing `>` isn't on a new line.

Reply:
without the blanks, clang-format now begins the line with `* spacedim>::`. I guess it is better than nothing.

Reply:
I can't express how much I loathe this tool.